packages/document/docs/zh/guide/basic/deploy.mdx

```diff
@@ -100,26 +100,26 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
Thanks! can you update the English version too?

Reply:
English version has also been added